Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -614,8 +621,8 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             stream <<< packageIdentity
             stream <<< manifestContents
             stream <<< toolsVersion.description
-            for key in env.keys.sorted(by: >) {
-                stream <<< key <<< env[key]! // forced unwrap safe
+            for (key, value) in env.sorted(by: { $0.key > $1.key }) {
+                stream <<< key <<< value
```

Comment:
This is a separate commit for getting rid of a force unwrap that I was while making these changes.  I can split out into a separate PR if needed (might be best)

Reply:
this is fine imo, small enough